registries/_format/time-local.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
For the name, I've seen programming languages call this a time only. I think it'd be interesting to see whether multiple languages use the same name (time only or otherwise) and check whether it's worth using a different name for clarity.

Reply:
I agree that given that `time` is already a format, `time-local` seems the most clear.  Unless you had a specific alternative in mind @baywet ?

Reply:
time-only was the alternative I was thinking about here. But I'm probably coming with [too much of a dotnet angle](https://learn.microsoft.com/en-us/dotnet/standard/datetime/how-to-use-dateonly-timeonly)

java calls it [local time](https://stackoverflow.com/questions/8384252/best-way-to-store-time-only-time-and-not-date-in-database)
go uses the same struct for everything

I guess I'm good with time-local, resolving.